### DISCUSSION TITLE
HDA Flowsheet Not Implemented Error

### DISCUSSION THREAD

**Discussion by JoshuaCNK** [IDAES/idaes-pse#1525] [Q&A]
Dear IDAES community,
I'm running into an error when going through the HDA Flowsheet and Optimization. When I run `seq.run(m, function)` , I get the following error 
`NotImplementedError: fs.H101.control_volume.properties_in property package has not implemented an initialize method. Please contact the property package developer ` . Can I have some assistance in how to fix this?

**Comment by dallan-keylogic** [comment]
What version of Python are you using? What version of IDAES? What version of IDAES Examples? This is the `hda_flowsheet_with_costing.ipynb` notebook, correct?

**Reply by JoshuaCNK** [reply]
I am using IDAES version 2.5, IDAES examples 2.6, and python version 3.10.15
I've been copying and pasting the code from the example online (https://idaes-examples.readthedocs.io/en/latest/docs/tut/core/hda_flowsheet_doc.html) into a jupyter notebook created via anaconda prompt in my virtual environment.

**Reply by dallan-keylogic** [reply]
Try updating your IDAES release to 2.6. The IDAES Examples are only guaranteed to work with an IDAES of the same version number.
